tests/controls/webtest.cpp

```diff
@@ -262,8 +269,71 @@ void WebTestCase::Zoom()
 
 void WebTestCase::RunScript()
 {
-    m_browser->RunScript("document.write(\"Hello World!\");");
+#if wxUSE_WEBVIEW_IE
```

Comment:
As mentioned above, I think this test is useless. A good test could verify that after setting the legacy/compatibility emulation level, running scripts returning objects doesn't work -- and, most importantly, when switching to the modern/standard mode it does work.